js/package.json

```diff
@@ -34,14 +34,14 @@
     "watch": "webpack -d -w"
   },
   "dependencies": {
-    "@jupyter-widgets/base": "^1.2.5 || ^2.0.0 || ^3.0.0 || ^4.0.0-alpha.2",
+    "@jupyter-widgets/base": "^1.2.5 || ^2.0.0 || ^3.0.0 || ^4.0.0",
     "bluebird": "^3.5.5",
     "jupyter-dataserializers": "^2.2.0",
     "three": "^0.97.0",
     "underscore": "^1.8.3"
   },
   "devDependencies": {
-    "@jupyterlab/builder": "^3.0.0-rc.13",
+    "@jupyterlab/builder": "^3.0.0",
```

Comment:
needs bump to at least `3.0.1`

Reply:
@bollwyvl what's the issue with the 3.0.0 release?

Reply:
The last time i was working down at this level, it complained to me when building... but i can't find where it applies the check in core...